File: RestBnb/Application/Bookings/Responses/BookingResponse.cs
using RestBnb.Core.Enums;
using System;

namespace RestBnb.Core.Contracts.V1.Responses
{
    public class BookingResponse
    {
        public int Id { get; set; }
        public DateTime CheckInDate { get; set; }
        public DateTime CheckOutDate { get; set; }
        public decimal PricePerNight { get; set; }
        public decimal TotalPrice { get; set; }
        public BookingState BookingState { get; set; }
        public int UserId { get; set; }
        public int PropertyId { get; set; }
    }
}

File: RestBnb/ClientApp/src/app/shared/models/bookingResponse.ts
export class BookingResponse {
  id: number;
  checkInDate: Date;
  checkOutDate: Date;
  pricePerNight: number;
  totalPrice: number;
  bookingState: BookingState;
  userId: number;
  propertyId: number;
  cancellationDate: Date | null;
}

export enum BookingState {
  Pending = 0,
  Accepted = 1,
}
